File: ProjetFilRouge/Models/Level.cs
﻿using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace ProjetFilRouge.Models
{
    public class Level
    {
        public int? IdLevel { get; set; }
        public string NameLevel { get; set; }
        public Level(int? idLevel,string nameLevel)
        {
            this.IdLevel = idLevel;
            this.NameLevel = nameLevel;
        }

        public Level() { }
    }
}

File: ProjetFilRougeFront/src/app/_models/level.ts
export class Level{
    private _idLevel: number;
    private _nameLevel: string;

    constructor(name: string = '', id: number = 0){
        this._idLevel = id;
        this._nameLevel = name;
    }

    get idLevel(): number{
        return this._idLevel;
    }

    get nameLevel(): string{
        return this._nameLevel;
    }

    set nameLevel(name: string){
        this._nameLevel = name;
    }
}